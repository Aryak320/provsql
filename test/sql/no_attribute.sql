\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

CREATE TABLE no_attribute AS SELECT FROM personnel;
SELECT remove_provenance('no_attribute');
SELECT * FROM no_attribute;
DROP TABLE no_attribute;
