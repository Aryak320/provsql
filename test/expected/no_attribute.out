\set ECHO none
remove_provenance

(1 row)

(7 rows)
